apis/kueue/v1beta1/admissioncheck_types.go

```diff
@@ -21,39 +21,43 @@ import (
 )
 
 const (
-	// The check cannot pass at this moment, back off (possibly
+	// CheckStateRetry means that the check cannot pass at this moment, back off (possibly
 	// allowing other to try, unblock quota) and retry.
 	// A workload having at least one check in the state,
 	// will be evicted if admitted will not be considered
 	// for admission.
 	CheckStateRetry = "Retry"
 
-	// The check will not pass in the near future. It is not worth
+	// CheckStateRejected means that the check will not pass in the near future. It is not worth
 	// to retry.
-	//NOTE: The admission behaviour is currently the same as for retry,
+	// NOTE: The admission behaviour is currently the same as for retry,
 	// we can consider marking the workload as "Finished" with a failure
 	// description.
 	CheckStateRejected = "Rejected"
 
-	// The state can be
+	// CheckStatePending means that the check still has been performed and the state can be
 	// 1. Unknown, the condition was added by kueue and its controller was not able to evaluate it.
 	// 2. Set by its controller and reevaluated after quota is reserved.
 	CheckStatePending = "Pending"
 
-	// The check has passed.
+	// CheckStateReady means that the check has passed.
 	// A workload having all its checks ready, and quota reserved can begin execution.
 	CheckStateReady = "Ready"
 )
 
 // AdmissionCheckSpec defines the desired state of AdmissionCheck
 type AdmissionCheckSpec struct {
-	// ControllerName identifies the controller managing this condition.
-	// +optional
+	// ControllerName is name of the controller which will actually perform
```

Comment:
```suggestion
	// controllerName is name of the controller which will actually perform
```
We should use the json name.

Reply:
Ah, that's right. Nice catch!
Done.